css/style.css

```diff
@@ -0,0 +1,81 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  font-family: Helvetica, Helvetica-Neue, Arial;
+  text-align: center;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: 1px solid #f1f3f3;
+  height: 450px;
+  width: 700px;
+}
+
+.snapshot {
+  height: 418px;
+  width: 313px;
+  display: inline-block;
+  margin: 15px;
+  border: 2px solid #f1f3f3;
+  border-radius: 10px;
+}
+
+.profile_pic {
+  width: 317px;
+  height: 250px;
+  overflow: hidden;
+  border: 0px;
+  border-radius: 10px 10px 0px 0px;
+  margin-left: -2px;
+  margin-top: -2px;
+  margin-bottom: 0px;
```

Comment:
you could put this definition on 1 line:
```css
margin: -2px 0 0 -2px;
```
but I actually don't think margin-bottom is necessary?